src/main/java/com/sleekbyte/tailor/utils/WhitespaceVerifier.java

```diff
@@ -75,4 +76,55 @@ public void verifyPunctuationIsSpaceDelimited(Token left, Token right, Token pun
                 + Messages.SPACE_AFTER, puncLocation);
         }
     }
+
+    /**
+     * Verifies if a parenthesized construct does not contain whitespace immediately after the opening parenthesis and
+     * immediately before the closing parenthesis.
+     *
+     * @param ctx Context comprised of parentheses
+     */
+    public void verifyParenthesisContentWhitespace(ParserRuleContext ctx) {
+        Location openingParenthesisLoc = ListenerUtil.getContextStartLocation(ctx);
+        Location closingParenthesisLoc = ListenerUtil.getContextStopLocation(ctx);
+
+        // Handles cases where the parentheses only contain whitespace
+        // Example: if ( ) {}
+        if (ctx.getChildCount() == 2) {
+            if (openingParenthesisLoc.line == closingParenthesisLoc.line
```

Comment:
you can use `checkLeftSpaces` or `checkRightSpaces` here instead of reimplementing the space counting logic